src/server/redis/caches.ts

```diff
@@ -446,3 +447,52 @@ export const dataForModelsCache = createCachedObject<ModelDataCache>({
     return results;
   },
 });
+
+type UserContentOverview = {
+  id: number;
+  modelCount: number;
+  imageCount: number;
+  videoCount: number;
+  postCount: number;
+  articleCount: number;
+  bountyCount: number;
+  bountyEntryCount: number;
+  hasReceivedReviews: boolean;
+  collectionCount: number;
+};
+
+export const userContentOverviewCache = createCachedObject<UserContentOverview>({
+  key: REDIS_KEYS.CACHES.OVERVIEW_USERS,
+  idKey: 'id',
+  lookupFn: async (ids) => {
+    const goodIds = ids.filter(isDefined);
+    if (!goodIds.length) return {};
+
+    const userOverviewData = await dbRead.$queryRaw<UserContentOverview[]>`
+    SELECT
+        u.id, 
+        (SELECT COUNT(*)::INT FROM "Model" m WHERE m."userId" = u.id AND m."status" = 'Published' AND m.availability != 'Private') as "modelCount",
+        (SELECT COUNT(*)::INT FROM "Post" p WHERE p."userId" = u.id AND p."publishedAt" IS NOT NULL AND p.availability != 'Private') as "postCount",
+        (SELECT COUNT(*)::INT FROM "Image" i
+          INNER JOIN "Post" p ON p."id" = i."postId" AND p."publishedAt" IS NOT NULL AND p.availability != 'Private'
```

Comment:
Are we keeping inner join here, or should we use left join?

Reply:
I tested this throughoutly, - by using LEFT it considers images with no post (which we don't exactly want since we don't show this), so we need Inner. I tried adding LEFT + `IS NOT NULL` but that literally resulted in the exact same performance as INNER.